frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedPanel.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { getActiveNamespace } from '@console/internal/actions/ui';
+import { ResourceIcon, ResourceLink, SimpleTabNav } from '@console/internal/components/utils';
+import { referenceFor } from '@console/internal/module/k8s';
+import TopologyOperatorBackedResources from './TopologyOperatorBackedResources';
+import { TopologyDataObject } from '../topology-types';
+
+export type TopologyResourcePanelProps = {
```

Comment:
```suggestion
export type TopologyOperatorBackedPanelProps = {
```
nit: can rename props based on component name

Reply:
Updated.